[Insights] Added the Tech preview information (#4332)
mesosphere/dcos-docs-site: pages/dkp/kommander/2.2/legal/version-policy/index.md

This software is provided "as is" and without any express or implied warranties including, without limitation, the implied warranties of merchantability and fitness for a particular purpose.

### Technical preview

> We provide Technical Preview, or Tech Preview, features to showcase capabilities that might be added to future versions of the product. As they are not yet production-ready, the support terms are the same as defined for experimental features. Technical Preview features are not guaranteed to move forward, so could be removed from future versions of the product.```

Reviewer: ```suggestion
We provide Technical Preview, or Tech Preview, features to showcase capabilities that might be added to future versions of the product. As they are not yet production-ready, the support terms are the same as defined for experimental features. Technical Preview features are not guaranteed to move forward, so could be removed from future versions of the product.

Reviewer: There are some extra backticks here it seems

This software is provided "as is" and without any express or implied warranties including, without limitation, the implied warranties of merchantability and fitness for a particular purpose.

### Technical preview

> Technical preview, or Tech Preview, is similar to Experimental status, but with the following differences:

Reviewer 2: ```suggestion
Technical Preview, or Tech Preview, is similar to Experimental status, but with the following differences:
```

Reviewer 2: Not sure if Preview should be capitalized here and below as other references are on the page?  Not title, but other references.

> Technical Preview, or Tech Preview, is similar to Experimental status, but with the following differences:

Reviewer 3: I think it's exactly the same as Experimental, and we should either copy pasta that section here, or add the words technical preview to the 'Experimental' section header.

> Technical Preview, or Tech Preview, is similar to Experimental status, but with the following differences:

Author: ```suggestion
Technical Preview, or Tech Preview, is similar to Experimental status, with the exception that Tech Preview items are considered part of the product and will be developed into future releases.
```

> - Tech Previews can be used in Production Environments.

Reviewer 3: I don't thing we want this here.     Keep in mind that the term 'tech preview' may apply to more than just insights over time.   And my understanding is that engineering is not agreeing to provide the current product (insights) with the SLA's necessary to support 'production environments'.   Certainly support is not.

> - Tech Previews can be used in Production Environments.

Author: ```suggestion
```

> - The licensing is subject to change in future versions.

Reviewer 3: I don't think licensing should be mentioned in the support terms.  They have nothing to do with each other.

> - The licensing is subject to change in future versions.

Author: ```suggestion
```
